File: code_challenge_solution/javascript/143 Pascal's triangle.js
```javascript
function generatePascalsTriangle(numRows) {
  let triangle = [];

  if (numRows === 0) {
    return triangle;
  }

  for (let i = 0; i < numRows; i++) {
    let row = [];
    for (let j = 0; j <= i; j++) {
      if (j === 0 || j === i) {
        row.push(1);
      } else {
        let prevRow = triangle[i - 1];
        let sum = prevRow[j - 1] + prevRow[j];
        row.push(sum);
      }
    }
    triangle.push(row);
  }

  return triangle;
}

console.log(generatePascalsTriangle(5));

```
Note: The parameter `numRows` indicates the number of rows in the triangle. Change the value of `numRows` to generate Pascal's triangle of different sizes.

File: code_challenge_solution/java/143 Pascal's triangle.java

class PascalTriangle {
    public static void main(String[] args) {
        int numRows = 5; // Specify the number of rows in Pascal's triangle

        for (int i = 0; i < numRows; i++) {
            int value = 1;
            for (int j = 0; j <= i; j++) {
                System.out.print(value + " ");
                value = value * (i - j) / (j + 1);
            }
            System.out.println();
        }
    }
}
